addon-templates/kubectl/dashboard/dashboard-controller.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   labels:
     k8s-app: kubernetes-dashboard
     kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
The addonmanager labels are needed when addon manager component is deployed to ensure the availability of the addons.